### ISSUE TITLE
templ_attr.cif: units of measurement in the _fract_matrix save frame

### ISSUE THREAD

**Issue by vaitkus** [COMCIFS/cif_core#207]
The units of measurement provided in the `_fract_matrix` save frame of the `templ_attr.cif`  file seem incorrect.

The `_fract_matrix` save frame describes elements of the matrix and vector that are used to transform Cartesian coordinates (in angstrom) to fractional coordinates (dimensionless) as described in the definition:
```
       x'                  |11 12 13|     x                  | 1 |
     ( y' )fractional = mat|21 22 23| * ( y )Cartesian +  vec| 2 |
       z'                  |31 32 33|     z                  | 3 |
```

The save frame lists `none` as the units, but this seems to only apply to the vector elements while the matrix elements should have the units of `reciprocal_angstroms`.

One way to resolve this would be to refactor the current save frame into two save frames (`_fract_matrix` and `_fract_vector`) and set the units accordingly to `reciprocal_angstroms` and `none`. Does this seems reasonable?

Also, for the sake of consistency, the same refactoring could be done to the `_Cartn_matrix` save frame that describes elements of the matrix and vector responsible for the reverse transformation (fractional to Cartesian), even though  both the matrix and the vector have the same units (see #206). Does this additional change also seems reasonable?

**Comment by jamesrhester** [comment]
Yes, this should be fixed, I think it was a cut-paste oversight.

**Comment by vaitkus** [comment]
Issue was resolved by merging PR #230.
